COPYING.md

```diff
@@ -64,24 +49,61 @@ This program is based on works distributed under the terms of another license(s)
     Copyright (c) 2008-2011  David G. Lowe (lowe@cs.ubc.ca). All rights reserved.
     Distributed under the [BSD License](http://www.opensource.org/licenses/bsd-license.php).
 
+*   __geogram__
```

Comment:
Depending on what you consider optional, this may be optional too.
If by optional you mean that eg there is an option in cmake to build without it then this is optional.
Otherwise, if optional means that it is a dependency that can just improve or change the performances of an algorithm (eg cxsparse) then this is not optional

Reply:
As SfM and MVS can be globally disabled, everything would be optional.
So I consider as non-optional dependencies the ones required to build the minimal SfM+MVS pipeline.